requirements.txt

```diff
@@ -1 +1 @@
-ansible>=2.2.0
+ansible<2.3
```

Comment:
It might be a fool's errand, but I don't want to be in the position of having to continually make changes to support the latest release.

It looks like Ansible releases about every 4 months, but supports the last two major releases.

http://docs.ansible.com/ansible/dev_guide/developing_releases.html

I guess this means we'll have to re-evaluate which versions we support about every 8 months?